lms/envs/load_test.py

```diff
@@ -20,4 +20,6 @@
 DEFAULT_TEMPLATE_ENGINE['OPTIONS']['context_processors'] = filter(
     EXCLUDE_CSRF, DEFAULT_TEMPLATE_ENGINE['OPTIONS']['context_processors']
 )
+INSTALLED_APPS.append('openedx.core.djangoapps.monitoring_utils.apps.MonitoringUtilsConfig')
+MIDDLEWARE_CLASSES.insert(0, 'openedx.core.djangoapps.monitoring_utils.middleware.MonitoringMemoryMiddleware')
```

Comment:
It will be installed in loadtest, but not actually capturing data unless the waffle switch is enabled.  I think the rationale was that we didn't want to possibly incur the additional database queries for checking the waffle switch in prod.